agent/util-scripts/pbench-postprocess-tools

```diff
@@ -92,6 +92,13 @@ fi
 tool_output_dir="$dir/tools-$group"
 mkdir -p $tool_output_dir
 
+# Try to prevent a cascade of tools running.
+if [ "$action" == "start" ] ;then
+	# Kill any tools running from a previous incantation before
+	# starting this one.
+	pbench-kill-tools --group $group
```

Comment:
I think this is a significant change to the PR.  Can you update the summary line for the commit message to somehow include this and then call this behavior change out explicitly in the commit message?